src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -193,6 +193,23 @@ export class SearchMenu extends React.Component {
     window.location.assign(searchUrl);
   };
 
+  formatSuggestion = suggestion => {
+    if (!suggestion || !this.state.userInput) {
+      return '';
```

Comment:
I think you should just return back the suggestion here? Or did you prefer to just omit the suggestion if it can't be formatted?